src/main/java/org/webcurator/domain/TargetInstanceDAOImpl.java

```diff
@@ -826,23 +819,44 @@ public Object doInHibernate(Session aSession) throws HibernateException {
 		private AbstractTarget target;
 		
 		private List<Schedule> schedules = new LinkedList<Schedule>();
-		
+
 		public UnscheduleTargetTransaction(AbstractTarget aTarget) {
 			target = aTarget;
 		}
-		
 		private void collectSchedules(AbstractTarget aTarget) {
+			Map<Object,Object> duplicateValidator=new HashMap<>();
+			collectSchedulesInterval(aTarget, duplicateValidator);
+			duplicateValidator.clear();
+		}
+		private void collectSchedulesInterval(AbstractTarget aTarget, Map<Object,Object> duplicateValidator) {
```

Comment:
Shouldn't the name be "collectSchedulesInternal"?

Reply:
Yes. It's collectSchedulesInternal. Fixed it.